packages/slice-machine/server/src/api/screenshots/custom-screenshots.ts

```diff
@@ -40,5 +41,8 @@ export default async function handler(
 
   Files.copy(file.path, pathToScreenshot, { recursive: true });
 
-  return createScreenshotUI(env.baseUrl, pathToScreenshot);
+  return createScreenshotUI(env.baseUrl, {
+    path: pathToScreenshot,
+    hash: hash(Files.readString(file.path)),
```

Comment:
yeah they're mainly the same files (we copied from lib to core at some point). I believe someone advocated for not using the core Files methods from inside slice-machine, as they could change, be renamed, etc. But it could be changed easily. Can you point me toward the engineering debt Notion page? I'll write something about it there

Reply:
posted it on the slack channel so every one else could get a look :)
there is a mention of duplicated code, if i remeber correctly `Files` was one of them